core/jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -72,15 +74,15 @@ export class CreateServiceComponent implements OnInit {
   model = new ServiceFormData('','','', '','','');
   cronObj = new CronObject('0/5','*','*','*','?','*')
   rateExpression = new RateExpression(undefined, undefined, 'none', '5', this.selected, '');
-  eventExpression = new EventExpression("awsEventsNone",undefined,undefined,undefined);
+  eventExpression = new EventExpression("awsEventsNone",undefined,undefined,undefined,undefined);
```

Comment:
why undefined?

Reply:
I think this is out of the scope of the current story. The EventExpression class should be using optional parameters or not be used at all. But we can decide that at a later date.

Reply:
can we create a task/issue/story to handle this separately